.gitignore

```diff
@@ -61,6 +61,10 @@ zconf-ng.h.cmakein
 ztest*
 /test/CTestTestfile.cmake
 /test/cmake_install.cmake
+/zconf-ng.h.included
```

Comment:
Would be cleaner to have this few lines above where similar filenames are excluded. Remove initial `/` to match style of other similar filenames.

Reply:
i'll do that